### PR TITLE
shell-completion: do not truncate suggestions

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: shell-completion/bash/busctl [hunk 1]
```diff
@@ -32,7 +32,7 @@ __get_machines() {
 __get_busnames() {
     local mode=$1
     local a b
-    busctl $mode list --no-legend --no-pager 2>/dev/null |
+    COLUMNS=1024 busctl $mode list --no-legend --no-pager 2>/dev/null |
```
Comment (keszybz) [inline shell-completion/bash/busctl]: Shouldn't this be 0 instead?
https://github.com/systemd/systemd/blob/master/src/basic/terminal-util.c#L800-L808

Comment (yuwata) [reply shell-completion/bash/busctl]: No, it does not work. `COLUMNS=0` is equivalent to not setting `COLUMNS=`.

Comment (keszybz) [reply shell-completion/bash/busctl]: Right. So what about USHRT_MAX ? It seems possible to exceed the 1024 limit easily.

Or even better, add a mode to busctl whihch does not truncate?

Comment (yuwata) [reply shell-completion/bash/busctl]: OK. Updated. Now `USHRT_MAX` is used. PTAL.

**Pull request merged**